authenticator/authenticator-api-public.yaml

```diff
@@ -260,15 +260,14 @@ definitions:
     properties:
       name:
         description: |
-          The name of the header in which the token should be presented on subsequent
-          API calls.
+          The name of the header in which the token should be presented on subsequent PI calls.
         type: string
       token:
         type: string
         description: |
           Token that can be passed as header, it can be considered a refresh token along with the
           idm/tokens ("authorizationToken").
-          This should be considered opaque data by the client. It is not intended to contain any data interpretable by the
+          This should be considered opaque data by the client. It is not intended to conatain any data interpretable by the
```

Comment:
typo here

Reply:
Is fixed here: https://github.com/SymphonyOSF/SBE/pull/23899